Revert back to old license syntax in order to drop the setuptools requirement.
author: micketeer@gmail.com

--- pyproject.toml.orig	2025-06-04 00:42:35.771720863 +0000
+++ pyproject.toml	2025-06-04 00:43:31.710935535 +0000
@@ -1,6 +1,6 @@
 [build-system]
 # require a recent setuptools for `license = ` support
-requires = ["setuptools >= 78.1.0", "wheel"]
+requires = ["setuptools >= 75.0.0", "wheel"]
 build-backend = "setuptools.build_meta"
 
 [project]
@@ -60,7 +60,7 @@
 
 There are also [optional extras available](https://hypothesis.readthedocs.io/en/latest/extras.html).
 """, "content-type" = "text/markdown"}
-license = "MPL-2.0"
+license = {text = "MPL-2.0"}
 requires-python = ">= 3.9"
 keywords = ["python", "testing", "fuzzing", "property-based-testing"]
 classifiers = [
